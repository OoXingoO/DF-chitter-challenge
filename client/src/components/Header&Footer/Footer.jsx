const Footer = () => {
    return (
        <div className="footer">
            <p>Â© 2022 Chitter</p>
        </div>
    )
}

export default Footer