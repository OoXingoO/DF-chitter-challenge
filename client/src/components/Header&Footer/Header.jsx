/* eslint-disable */
import logo from '../../utils/logo.png'

const Header = () => {
    return (
        <nav>
            <ul className='topnav'>
                <li className='logo'><a href="#">
                    <img className="img-fluid" src={logo} alt="Chitter Logo" />
                </a>
                </li>
                <li id='home-item' ><a className="home" href="#">Home</a></li>
                <li id='logout-item' className='logOut'><a className='logOut-button' href='#'>Log out</a></li>
            </ul>
        </nav>
    )
}

export default Header;