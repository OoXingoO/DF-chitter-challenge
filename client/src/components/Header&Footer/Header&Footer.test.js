import { render, screen } from '@testing-library/react';
import Header from './Header';
import Footer from './Footer';

test(`Header matches snapshot`, () => {
    const headerComponent = render(<Header />);

    expect(headerComponent).toMatchSnapshot();
});

test(`Footer matches snapshot`, () => {
    const footerComponent = render(<Footer />);

    expect(footerComponent).toMatchSnapshot();
});