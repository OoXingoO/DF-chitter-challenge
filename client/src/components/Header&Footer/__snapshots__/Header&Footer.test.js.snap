// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="footer"
      >
        <p>
          © 2022 Chitter
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="footer"
    >
      <p>
        © 2022 Chitter
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Header matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav>
        <ul
          class="topnav"
        >
          <li
            class="logo"
          >
            <a
              href="#"
            >
              <img
                alt="Chitter Logo"
                class="img-fluid"
                src="logo.png"
              />
            </a>
          </li>
          <li
            id="home-item"
          >
            <a
              class="home"
              href="#"
            >
              Home
            </a>
          </li>
          <li
            class="logOut"
            id="logout-item"
          >
            <a
              class="logOut-button"
              href="#"
            >
              Log out
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav>
      <ul
        class="topnav"
      >
        <li
          class="logo"
        >
          <a
            href="#"
          >
            <img
              alt="Chitter Logo"
              class="img-fluid"
              src="logo.png"
            />
          </a>
        </li>
        <li
          id="home-item"
        >
          <a
            class="home"
            href="#"
          >
            Home
          </a>
        </li>
        <li
          class="logOut"
          id="logout-item"
        >
          <a
            class="logOut-button"
            href="#"
          >
            Log out
          </a>
        </li>
      </ul>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
